app/controllers/api/items_controller.rb

```diff
@@ -1,23 +1,22 @@
 class Api::ItemsController < Api::ApiController
-
   def sync_manager
-    if !@sync_manager
+    unless @sync_manager
       version = params[:api]
-      if version == "20190520"
-        @sync_manager = SyncEngine::V20190520::SyncManager.new(current_user)
+      @sync_manager = if version == '20190520'
```

Comment:
Should the if statement here be on the next line?

Reply:
Yes, it works too:

```
@sync_manager =
  if version == '20190520'
    SyncEngine::V20190520::SyncManager.new(current_user)
  else
    SyncEngine::V20161215::SyncManager.new(current_user)
  end
```